src/util.ts

```diff
@@ -1,14 +1,90 @@
 import * as grpc from "@grpc/grpc-js";
+import { ConnectionOptions } from 'tls';
+
+
+// NOTE: Copied from channel-credentials.ts in gRPC Node package because its not exported.
+class ComposedChannelCredentialsImpl extends grpc.ChannelCredentials {
+  constructor(
+    private channelCredentials: KnownInsecureChannelCredentialsImpl,
+    callCreds: grpc.CallCredentials
+  ) {
+    super(callCreds);
+  }
+  compose(callCredentials: grpc.CallCredentials) {
+    const combinedCallCredentials = this.callCredentials.compose(
+      callCredentials
+    );
+    return new ComposedChannelCredentialsImpl(
+      this.channelCredentials,
+      combinedCallCredentials
+    );
+  }
+
+  _getConnectionOptions(): ConnectionOptions | null {
+    return this.channelCredentials._getConnectionOptions();
+  }
+  _isSecure(): boolean {
+    return false;
+  }
+  _equals(other: grpc.ChannelCredentials): boolean {
+    if (this === other) {
+      return true;
+    }
+    if (other instanceof ComposedChannelCredentialsImpl) {
+      return (
+        this.channelCredentials._equals(other.channelCredentials) &&
+        this.callCredentials._equals(other.callCredentials)
+      );
+    } else {
+      return false;
+    }
+  }
+}
+
+// Create our own known insecure channel creds.
```

Comment:
Add a note that this is for testing only

Reply:
Well, some people might not use it for testing if they are within a VPC